Marlin/planner.h

```diff
@@ -166,6 +166,11 @@ class Planner {
      * Nominal speed of previous path line segment
      */
     static float previous_nominal_speed;
+	
+	/**
+	 * Limit where 64bit math is necessary for acceleration calculation
+	 */
+	static unsigned long cutoff_32bit;
```

Comment:
Actually, this is just following the established pattern. `Planner` is a `static` singleton class. All the variables in `planner.cpp` are data members. We have no global statics set up in `planner.cpp`. There's actually no special advantage to making the variable a root-namespace global within `planner.cpp`. If the variable doesn't need to be seen by other sources, it should just be in the `private` section of the class.